variables.go

```diff
@@ -347,3 +317,41 @@ func (r *varReplacer) replaceVars(vars Vars) Vars {
 	}
 	return new
 }
+
+var (
+	templateFuncs template.FuncMap
+)
+
+func init() {
+	taskFuncs := template.FuncMap{
+		"OS":   func() string { return runtime.GOOS },
+		"ARCH": func() string { return runtime.GOARCH },
+		"catLines": func(s string) string {
+			s = strings.Replace(s, "\r\n", " ", -1)
+			return strings.Replace(s, "\n", " ", -1)
+		},
+		"splitLines": func(s string) []string {
+			s = strings.Replace(s, "\r\n", "\n", -1)
+			return strings.Split(s, "\n")
+		},
```

Comment:
👆

Reply:
If you prefer the `PascalCase`, I can rename them, or if you prefer consistency with `sprig`, we can rename all of them, but leave backwards compatibility:

```go
 	taskFuncs := template.FuncMap{
		(...)
		"fromSlash": func(path string) string {
			return filepath.FromSlash(path)
		},
		"toSlash": func(path string) string {
			return filepath.ToSlash(path)
		},
		"exeExt": func() string {
			if runtime.GOOS == "windows" {
				return ".exe"
			}
			return ""
		},
		// IsSH is deprecated.
		"IsSH": func() bool { return true },
	}
 	// Deprecated aliases for renamed functions.
	task["FromSlash"] = taks["fromSlash"]
	task["ToSlash"] = taks["toSlash"]
	task["ExeExt"] = taks["extExt"]
```

We can comment the backwards compatibility only very briefly in the README.

Reply:
Agree with renaming them. The reason this is not consistent is because spring was added after some of the other funcs.

Reply:
Yeah, I assumed it was historical reasons:-)